lib/smack/SmackRep.cpp

```diff
@@ -32,6 +39,9 @@ const Expr* SmackRep::NUL = Expr::id(NULL_VAL);
 const string SmackRep::STATIC_INIT = "$static_init";
 
 const int SmackRep::width = 0;
+const unsigned SmackRep::ptrsize = 32;
```

Comment:
What would happen if we queried `llvm::DataLayout` instead of hard-coding this 32 bits? http://llvm.org/docs/doxygen/html/classllvm_1_1DataLayout.html#a9e653935f1d7ff84ff4a14667f4ca567